style.css

```diff
@@ -0,0 +1,478 @@
+  background-color: chocolate;
+}
+
+/* Dropdown content (hidden by default) */
+.dropdown-categories,
+.dropdown-sorting,
+.dropdown-vegetarian {
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+  z-index: 1;
+}
+
+/* Links inside the dropdown */
+.dropdown-categories a,
+.dropdown-sorting a,
+.dropdown-vegetarian a {
+  float: none;
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+  text-align: left;
+}
+
+/* Add a grey background color to dropdown links on hover */
+.dropdown-categories a:hover,
+.dropdown-sorting a:hover,
+.dropdown-vegetarian a:hover {
+  background-color: #ddd;
+}
+
+/* Show the dropdown menu on hover */
+.dropdown1:hover .dropdown-categories,
+.dropdown2:hover .dropdown-sorting,
+.dropdown3:hover .dropdown-vegetarian {
+  display: block;
+}
+.fa {
+  color: rgb(236, 138, 68);
+}
+
+.min-max-slider {
+  position: relative;
+  width: 200px;
+  text-align: center;
+  margin-bottom: 50px;
+  margin-left: 300px;
+  padding-top: 11px;
+}
+
+.min-max-slider > label {
+  display: none;
+}
+
+/* Slider */
+span.value {
+  height: 1.5em;
+  font-weight: bold;
+  margin-right: 117px;
+  font-size: 15px;
+  text-align: center;
+  margin-bottom: 20px;
+  color: rgba(0, 0, 0, 0.404);
+}
+
+span.value.lower::before {
+  content: "$";
+  margin-left: -25px;
+  color: rgba(0, 0, 0, 0.404);
+}
+
+span.value.upper::before {
+  content: "$";
+  margin-left: 0.5em;
+  padding-top: 5px;
+  padding-left: 90px;
+  color: rgba(0, 0, 0, 0.404);
+  font-weight: bold;
+}
+
+.min-max-slider > .legend {
+  display: flex;
+  justify-content: space-between;
+}
+
+.min-max-slider > .legend > * {
+  font-size: small;
+  opacity: 0.25;
+}
+
+.min-max-slider > input {
+  cursor: pointer;
+  position: absolute;
+}
+
+/* webkit specific styling */
+.min-max-slider > input {
+  -webkit-appearance: none;
+  outline: none !important;
+  background: transparent;
+  background-image: linear-gradient(
+    to bottom,
+    transparent 0%,
+    transparent 30%,
+    rgb(236, 138, 68) 30%,
+    rgb(236, 138, 68) 60%,
+    transparent 60%,
+    transparent 100%
+  );
+}
+
+.min-max-slider > input::-webkit-slider-thumb {
+  -webkit-appearance: none; /* Override default look */
+  appearance: none;
+  width: 14px; /* Set a specific slider handle width */
+  height: 14px; /* Slider handle height */
+  background: #eee; /* Green background */
+  cursor: pointer; /* Cursor on hover */
+  border: 1px solid rgb(236, 138, 68);
+  border-radius: 100%;
+}
+
+.min-max-slider > input::-webkit-slider-runnable-track {
+  cursor: pointer;
+}
+
+h6 {
+  text-align: left;
+  margin-left: -75px;
+  margin-top: -18px;
+  font-size: 15px;
+  color: rgba(0, 0, 0, 0.404);
+  font-weight: bold;
+}
+
+/* Product */
+.image {
+  height: 170px;
+  width: auto;
+  margin-top: 0px;
+  margin-left: 60px;
+  border-radius: 15px;
+}
+
+.image1 {
```

Comment:
This is not a good naming for this class. Avoid using classes and methods like this image1, method1, method3, etc